src/bin/pgcopydb/ld_stream.c

```diff
@@ -1237,6 +1199,28 @@ streamCloseFile(LogicalStreamContext *context, bool time_to_abort)
 		}
 	}
 
+	/*
+	 * ROLLBACK transactions that are still in progress when we abort.
+	 * When resuming, we always start from a consistent point, so we don't need
+	 * worry about partial transactions which are rollbacked here.
+	 */
```

Comment:
What happens in case of a crash? I think we need a short reference to that in the comment (the code to handle restart after crash might not be here, but rather where we read a JSON file from disk and resume from it).